modules/Link.js

```diff
@@ -79,12 +87,15 @@ class Link extends React.Component {
     } = this.props
 
     const currentLocation = location || this.context.location
+    let isActive = false
```

Comment:
`const isActive = router ? getIsActive(...) : false`

I think avoiding a `let` is worth a ternary, but idc that match, do it however you like.